analysis/make_config.py

```
@@ -0,0 +1,59 @@
+            compare_with_previous_s_2_1 = True
+            compare_with_previous_s_2_2 = True
+            compare_with_history_s_2_1 = True
+            compare_with_history_s_2_2 = True
+
+            # 其他配置...
+            
+
+        class ActionSQLConfig:
+            sql = f"""
+            with users as (
+                select distinct {MarketingUsersConfig.id_field} as ccif_no
```

Ruff (0.8.2)

Undefined name `MarketingUsersConfig`

(F821)